src/core/test_helpers/kbn_server.ts

```diff
@@ -247,6 +247,7 @@ export function createTestServers({
       return {
         stop: async () => await es.cleanup(),
         es,
+        // TODO: Need to support multiple URLs here
```

Comment:
These updates will need to be made pending a decision on what to do about `es_test_config` in `@kbn/test`